src/util/KeyboardNavigation.ts

```diff
@@ -0,0 +1,93 @@
+import { type ArrowKeyCode, KeyCode } from './KeyCode';
+import { arrayMinByKey, getActiveElement, isHTMLElement } from './CommonUtils';
+import { Rectangle } from './GeometryUtils';
+
+export function getFocusedChild(children: HTMLElement[]): HTMLElement | null {
+    const focusedChild = getActiveElement();
+    if (!focusedChild || focusedChild === document.body || !isHTMLElement(focusedChild)) {
+        if (children.length > 0) {
+            children[0].focus();
```

Comment:
It's a bit weird to *change* the `focus()` inside a function that seemingly only *retrieves* a focused element... 😕

Can we move this out somehow?